src/components/Nav/index.js

```diff
@@ -1,7 +1,17 @@
 import React, {useState} from 'react';
-import styled, {css} from 'styled-components/macro';
+
 import {FaAlignJustify, FaBackspace} from 'react-icons/fa';
 
+import categories from '../../data/category.json';
+import MenuItem from 'components/Nav/MenuItem';
+import {
+    HeaderNav,
+    SidebarSection,
+    CloseButton,
+    MenuList,
+    MenuTittle,
+} from './styles';
```

Comment:
Not really sure about the distinction between styles and components.. components have styles, too. Also `SidebarSection` has some custom logic.

Reply:
i just seperated. i would like to try it